model/src/main/kotlin/OrtResult.kt

```diff
@@ -33,12 +34,11 @@ import org.ossreviewtoolkit.spdx.SpdxExpression
 import org.ossreviewtoolkit.utils.log
 import org.ossreviewtoolkit.utils.zipWithDefault
 
-typealias CustomData = MutableMap<String, Any>
-
 /**
  * The common output format for the analyzer and scanner. It contains information about the scanned repository, and the
  * analyzer and scanner will add their result to it.
  */
+@JsonIgnoreProperties(value = ["data"])
```

Comment:
Could be just `@JsonIgnoreProperties("data")`.

Reply:
Done.